### DISCUSSION TITLE
Data lost or corrupted ?

### DISCUSSION THREAD

**Discussion by jmlaurenti** [Budibase/budibase#7842] [Help] [answered]
Hi all, 
this morning when accessing my budibase instance hosted in Kubernetes, I'm facing a screen I never saw before, asking me to Set budibase organisation.
Whatever data I enter, next screen is the login screen, but I'm unable to sign, always ending with "User not found " popup. :-(

Of course (would be too simple), I didn't setup yet the backup procedure as I'm still playing with the tool, 

I'm using v1.3.11, and all updates so far were successfull.

Any idea of the root cause ? CouchDb ? MiniO ? 

Thanks !
JM

**Comment by shogunpurple** [comment]
@jmlaurenti did you update your Budibase installation or perform any config on the kubernetes side? Or are you saying as soon as you accessed the platform this morning, something changed overnight?

**Comment by jmlaurenti** [comment]
Hi @shogunpurple , 
thanks for your response !
This morning I forced a restart on kubernetes (using argoCD) to get most recent version, so it's probably me who did something wrong.
Since them, I found error logs in CouchDB pod stating the users database was missing.
I was able to connect to CouchDB using Fauxton and a local kubernetes port forward, and saw users table is indeed missing.

That's where I am :-(

**Reply by shogunpurple** [reply]
hi @jmlaurenti - couchDB should not have changed between the updates, so I'm not sure that's the issue. Are all of your images (app service, worker service, and proxy service) all definitely up and running on the latest version of budibase?

With argoCD, are you just using the standard packaged chart and did you just update to have the latest version, then redeployed with the same configuration settings? 

Or do you have our YAML files directly in your github repo that you pulled originally using `helm pull` or similar, then managed by argo on git pushes? If the latter, your YAML files could be outdated.

**Comment by jmlaurenti** [comment, marked as answer]
ok, so, I don't know what happened to have  Users table corrupted, burt here is what I did to fix my issue:
- I created users table manually (it stopped the error logs in couchdb)
- Accessed again the "Set budibase organisation" screen, but this time entered "default"
- Then I was prompted with the normal login page, and was finally able to successfully use my login !

I certainly did a mistake when restarting in Argo, as I found trace of a deleted pod, but main lesson is that:
- data was still there
- in self hosting, organisation name is "default"

Thanks again for your help @shogunpurple  ! Sorry to have taken some of your time for my own mistake !

JM
